src/components/social/_whatsapp.less

```diff
@@ -0,0 +1,12 @@
+.social-btn {
+  &.whatsapp {
+    background-color: #25d366;
+    color: #fff;
+
+    &:after {
+      color: white;
+      right: 0px !important;
```

Comment:
this should be inside `.whatsapp.block-btn:after` something like this:
```
&.block-btn {
&:after {
right: 0px !important;
}
}
```

Reply:
Please check and review..